test/package/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+set_property(DIRECTORY APPEND
+		PROPERTY LABELS package
+)
+
+set(test_list
+		FetchContent
+		find_package
+)
+if (NOT Template_IS_TOP_LEVEL)
+	list(APPEND test_list pkg-config)
+endif ()
```

Comment:
Packaging tests, also callable from `tmt`. `pkg-config` I don't think can be easily tested outside of a `tmt` environment because of the `configure_file` format.